eclipse/common-dependencies.tpd

```diff
@@ -0,0 +1,23 @@
+/*
+ * Common Dependencies for the different platform Target Platform Dependencies.
+ */
+target "Cloud Tools for Eclipse: Common Dependencies"
```

Comment:
Is the file will be included as-is? If so, is it OK to have multiple `target`s defined?

Reply:
Yeah it's fine.  The .target files have all been generated from this.  I have another tweak coming to simplify the generation.